e2eshark/run.py

```diff
@@ -331,8 +331,13 @@ def runCodeGeneration(
         print(f"Test {testName} failed [{curphase}]")
         return 1
     logfilename = curphase + ".log"
+    # use same compile command we do in turbine
     commandname = (
-        "/tools/iree-compile --iree-input-demote-i64-to-i32 --iree-hal-target-backends="
+        "/tools/iree-compile --iree-input-type=torch --mlir-print-debuginfo "
+        + "--mlir-print-op-on-diagnostic=false --iree-llvmcpu-target-cpu-features=host "
+        + "--iree-llvmcpu-target-triple=x86_64-linux-gnu --iree-stream-resource-index-bits=64 "
```

Comment:
Would prefer to not hardcode this script to only run on Linux :)

In the IREE repo we also have benchmarks running with both default flags and with 'experimental' flags that are still being evaluated. Could do something similar here. Tests should use default flags wherever possible.

Reply:
Ok, will revert back to original options. Don't think they're necessary, just wanted consistency between turbine and here. But, default flags should do the trick. (I don't think we should be messing too much with the backend configurations here anyways)